<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegForm</class>
 <widget class="QWidget" name="RegForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>551</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: italic 22pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>Le mots</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: italic 16pt &quot;Sans Serif&quot;; background-color: rgb(230, 211, 250);</string>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: italic 16pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>   Введите логин</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="logEdit">
      <property name="styleSheet">
       <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
border :  2px solid rgb(132, 82, 206);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: italic 16pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>   Введите пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pasEdit">
      <property name="styleSheet">
       <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
border :  2px solid rgb(132, 82, 206);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font:  italic 16pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>   Повторите пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pasRepitEdit">
      <property name="styleSheet">
       <string notr="true">font: italic 14pt &quot;Sans Serif&quot;;
border :  2px solid rgb(132, 82, 206);</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(230, 211, 250);
font: italic 18pt &quot;Sans Serif&quot;;
</string>
      </property>
      <property name="text">
       <string>Зарегистрироваться!</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
